spectral/README.md

```diff
@@ -21,10 +21,16 @@ echo "extends: '@rhoas/spectral-ruleset'" > .spectral.yaml
 Run the Spectral CLI:
 
 ```shell
-npm install -g @stoplight/spectral@5.9.0
+npm install -g @stoplight/spectral-cli@6.4.1
```

Comment:
I'm wondering if we can wrap rules and cli into single package with locked versioning that we control. 
There would be multiple benefits from that:

- .spectral.yaml would not be needed
- We will use and control right version of the cli
- We will position linter installing locally as opposed to global install (which is antipattern for tools that strongly depend on versions)